SDL/Module.h

```diff
@@ -137,18 +137,18 @@ namespace SDL {
           device_nTotalSegs_buf(allocBufWrapper<unsigned int>(devAccIn, 1, queue)),
           device_nTotalTrips_buf(allocBufWrapper<unsigned int>(devAccIn, 1, queue)),
           device_nTotalQuints_buf(allocBufWrapper<unsigned int>(devAccIn, 1, queue)) {
-      alpaka::memset(queue, hitRanges_buf, -1);
-      alpaka::memset(queue, hitRangesLower_buf, -1);
-      alpaka::memset(queue, hitRangesUpper_buf, -1);
-      alpaka::memset(queue, hitRangesnLower_buf, -1);
-      alpaka::memset(queue, hitRangesnUpper_buf, -1);
-      alpaka::memset(queue, mdRanges_buf, -1);
-      alpaka::memset(queue, segmentRanges_buf, -1);
-      alpaka::memset(queue, trackletRanges_buf, -1);
-      alpaka::memset(queue, tripletRanges_buf, -1);
-      alpaka::memset(queue, trackCandidateRanges_buf, -1);
-      alpaka::memset(queue, quintupletRanges_buf, -1);
-      alpaka::memset(queue, quintupletModuleIndices_buf, -1);
+      alpaka::memset(queue, hitRanges_buf, 0xff);
+      alpaka::memset(queue, hitRangesLower_buf, 0xff);
+      alpaka::memset(queue, hitRangesUpper_buf, 0xff);
+      alpaka::memset(queue, hitRangesnLower_buf, 0xff);
+      alpaka::memset(queue, hitRangesnUpper_buf, 0xff);
+      alpaka::memset(queue, mdRanges_buf, 0xff);
+      alpaka::memset(queue, segmentRanges_buf, 0xff);
+      alpaka::memset(queue, trackletRanges_buf, 0xff);
+      alpaka::memset(queue, tripletRanges_buf, 0xff);
+      alpaka::memset(queue, trackCandidateRanges_buf, 0xff);
+      alpaka::memset(queue, quintupletRanges_buf, 0xff);
+      alpaka::memset(queue, quintupletModuleIndices_buf, 0xff);
```

Comment:
For some of these the warning makes sense, but again I think it might be better to remove them or set them to either `~0` or `0xffffffff` since they are 32-bit integers

Reply:
that's what memset does here, it sets the values to `0xffffffff` for the destination types.
The source is a byte
https://alpaka-group.github.io/alpaka/namespacealpaka.html#aa7b4d9f42d421578d38e8923da784b37
`Sets the bytes of the memory of view to the given value.`

Reply:
Same comment as above.